test/dotnet-run.Tests/RunFileTests.cs

```diff
@@ -300,7 +297,7 @@ public void ClassLibrary_EntryPointFileExists()
             .WithWorkingDirectory(testInstance.Path)
             .Execute()
             .Should().Fail()
-            .And.HaveStdErrContaining(s_noTopLevelStatements);
+            .And.HaveStdOutContaining("error CS5001:"); // Program does not contain a static 'Main' method suitable for an entry point
```

Comment:
Is the word "error" localized? Meaning, would this word change if it was run in a different language environment?

Reply:
Yes, it should be. Note that this is the output of the Roslyn compiler. Not sure why you are asking? It doesn't look like sdk tests run in different locales and there are many tests that verify localizable strings against their English variant. But for example, in Roslyn tests, we don't do that as we run the tests also on non-English locales intentionally. If you intend to do that as well in sdk in the future, let me know, I will try to rewrite my tests so they are locale-insensitive.

Reply:
Some people run SDK tests locally. They would get localized outputs for this sort of thing, so this test wouldn't work for them as-is.

Reply:
Makes sense, I can follow up with a PR that ensures RunFileTests and DotnetProjectConvertTests can run on localized machines.

But I want to reiterate that this is a pre-existing problem for many other tests, e.g., https://github.com/search?q=repo%3Adotnet%2Fsdk%20HaveStdErrContaining&type=code